### ISSUE TITLE
Wrong links behind `Recordings` button in Past Events section

### ISSUE THREAD

**Issue by niklasmtj** [open-gitops/website#91]
Right now it seems that when a new event joins the "Past Events" section the `Recordings` button does not change the link's `href`. This causes that for example the _GitOpsDays_ event holds the link to the _GitOpsCon_ YouTube playlist which can be seen in the picture below.

<img width="1512" alt="Bildschirmfoto 2022-06-09 um 17 00 52" src="https://user-images.githubusercontent.com/22395665/172879301-fa91d098-6b86-463d-8356-b003020de3b3.png">

I think we need a little more logic in the [event.js](https://github.com/open-gitops/website/blob/main/src/components/events.js) file which only shows an recordings button when there is an url set for the item. It seems that [line 95](https://github.com/open-gitops/website/blob/ca2948555e5ba0674a511a641409080d60f935c5/src/components/events.js#L95) gets the wrong url.

**Comment by scottrigby** [comment]
This specific example was just fixed in #95 

But @niklasmtj I agree, and have thought we could perhaps change our metadata keys to be more explicit, like:
```diff
---
title:
date:
location:
- url: 
+ attend:
+ recordings:
---
```
and in this way, if the key is missing or left blank then the corresponding "Attend" or "Recordings" button would not display. WDYT?

**Comment by niklasmtj** [comment]
Sounds like a good idea! The only thing that I see here would be that people don't know the `recordings` link in advance. So this way we have to be proactive to remind them to either open a follow up PR after the event or sending us the link so we will open the PR updating the `recordings` field afterwards. 
Either way this shouldn't be much of an issue and would make sure that we offer the recordings button only to the events where one is available 👍🏼
